docs: Update README for ImageProvider (#1037)
fluttercandies/flutter_photo_manager: README.md

#### Display assets

Since v3.0.0, the plugin doesn't provide any UI widgets.

> The original `AssetEntityImage` was moved to the new package [`photo_manager_image_provider`][photo_manager_image_provider].

Reviewer: ```suggestion
`AssetEntityImage` and `AssetEntityImageProvider` are provided in the new package
[`photo_manager_image_provider`][photo_manager_image_provider].
```
